Dockerfile

```
@@ -0,0 +1,68 @@
+# Taken and adjusted from https://github.com/vercel/next.js/blob/canary/examples/with-docker/Dockerfile
+
+FROM node:18-alpine AS base
```

```suggestion
FROM node:20-alpine AS base
```

it's possible to sync that with `.nvmrc` ?

---

yes thanks, should be the same now